README.md

```diff
@@ -20,6 +20,13 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+make examples-native
+cd examples/hello_triangle_c/build/
+./hello_triangle
```

Comment:
does it have to run from the `build` folder? maybe we can avoid `cd` altogether?

Reply:
Alternatively during the build we could include the shaders or set an absolute path for them